package.json

```diff
@@ -6,6 +6,11 @@
   ],
   "scripts": {
     "build": "tsc -b tsproject.json",
+    "build-core": "tsc -p packages/core/tsconfig.json",
+    "build-fs": "tsc -p packages/fs/tsconfig.json",
+    "build-imports": "tsc -p packages/imports/tsconfig.json",
+    "build-imports-fs": "tsc -p packages/imports-fs/tsconfig.json",
+    "build-ordered": "yarn build-core && yarn build-fs && yarn build-imports && yarn build-imports-fs",
```

Comment:
also check out `wsrun --stages build`

Reply:
@cymerrad TL;DR; your builds didn't work because you tried to do tests without compiled projects. Hypothesis: if you drop all those build scripts and just use `"test": "run-s build test:execute', "test:execute": "wsrun -p tests test"` it'll work